src/main/java/uk/gov/hmcts/darts/transcriptions/validator/WorkflowValidator.java

```diff
@@ -0,0 +1,76 @@
+package uk.gov.hmcts.darts.transcriptions.validator;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.darts.transcriptions.enums.TranscriptionStatusEnum;
+import uk.gov.hmcts.darts.transcriptions.enums.TranscriptionTypeEnum;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
+import static uk.gov.hmcts.darts.transcriptions.enums.TranscriptionStatusEnum.APPROVED;
+import static uk.gov.hmcts.darts.transcriptions.enums.TranscriptionStatusEnum.AWAITING_AUTHORISATION;
+import static uk.gov.hmcts.darts.transcriptions.enums.TranscriptionStatusEnum.CLOSED;
+import static uk.gov.hmcts.darts.transcriptions.enums.TranscriptionStatusEnum.COMPLETE;
+import static uk.gov.hmcts.darts.transcriptions.enums.TranscriptionStatusEnum.REJECTED;
+import static uk.gov.hmcts.darts.transcriptions.enums.TranscriptionStatusEnum.REQUESTED;
+import static uk.gov.hmcts.darts.transcriptions.enums.TranscriptionStatusEnum.WITH_TRANSCRIBER;
+import static uk.gov.hmcts.darts.transcriptions.enums.TranscriptionTypeEnum.OTHER;
+
+@Component
+@Slf4j
+public class WorkflowValidator {
+
+    // Create a map defining the transition rules, from any given current state to a list of allowable target states
+    private final Map<TranscriptionStatusEnum, Set<TranscriptionStatusEnum>> manualWorkflowTransitionRules = new HashMap<>();
+
+    private final Map<TranscriptionStatusEnum, Set<TranscriptionStatusEnum>> automaticWorkflowTransitionRules = new HashMap<>();
+
+    public WorkflowValidator() {
+        manualWorkflowTransitionRules.put(REQUESTED, Set.of(REQUESTED, AWAITING_AUTHORISATION, CLOSED));
```

Comment:
It doesn't look right to have REQUESTED on the `to` side - don't follow that bit

Reply:
The first time you save a transcription, its current state will be requested. At the time I thought the validation would be tied to the workflow save method so it would go from current state REQUESTED to REQUESTED, but if its only going in the update, that can be removed.

Reply:
No, I think we should remove it because the code is ensuring that both Transcription and TranscriptionWorkflow status is kept in sync - saved in the same transaction. It's also thought that the database trigger that Barry introduced recently will be dropped in the next version of the schema.

Reply:
done